### PR TITLE
[SQL] [Lua] Add "Turkey with Rolanberry Sauce" food issue #943

### PR DESCRIPTION
**Pull request by CatsEyeXI**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Closes: #943
^ Making Rolanberry Turkey usable.

## Steps to test these changes

1. Check attributes
2. !additem 6576
3. Use Item
4. Check attributes

I am not aware of how to check my current "Counter" and "Amnesia Resist" but assuming these mods work there should be no problem here. 
STR, Attack and Ranged Attack confirmed.

### CODE REVIEW

File: scripts/globals/items/turkey_with_rolanberry_sauce.lua [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+-- https://www.bg-wiki.com/ffxi/Rol._Turkey
+-----------------------------------
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    local result = 0
+    if (target:hasStatusEffect(xi.effect.FOOD) or target:hasStatusEffect(xi.effect.FIELD_SUPPORT_FOOD)) then
```
Comment (Xaver-DaRed) [inline scripts/globals/items/turkey_with_rolanberry_sauce.lua]: Remove the outer parenthesis

**Comment by CatsEyeXI** [comment]
looks like startup check failed because port was already in use

**Review by Xaver-DaRed — APPROVED** [review]
Item cannot be easily captured, so I'm OK with the TODO

**Comment by zach2good** [comment]
Teo also agrees in the other issue:
> Do foods have diff use animations? I'd never noticed/paid attention. Even so, you can get away with copying the value from a similar food and putting a todo comment if the animation is wrong. I have a couple of these and when I finally use them, I can fix it if nobody else has.

**Pull request merged**